src/robotide/editor/kweditor.py

```diff
@@ -568,24 +567,23 @@ def OnKeyDown(self, event):
             if keycode == wx.WXK_SPACE:
                 self._open_cell_editor_with_content_assist()  # Mac CMD
             elif specialkcode in [wx.WXK_DOWN, wx.WXK_UP]:
-                self._skip_except_on_mac(event)
```

Comment:
put this back, move_rows doesn't work on windows

Reply:
I only tested this PR in Mac, let me test more in windows and linux.

Reply:
@Nyral  tested in windows and linux, move row function works well, that line can be removed.

Reply:
Interesting... did you test in Windows 10 or 7?

Reply:
yes, win7 & win10 with the combination of wxPython 4.0.7.post2 & wxPython 4.1.0

Reply:
@HelioGuilherme66 Does it work for you?

Reply:
Ok. If it doesn't work then we can always find a fix for it in the future.